kenya_compliance/kenya_compliance/overrides/server/stock_ledger_entry.py

```diff
@@ -271,17 +291,13 @@ def get_stock_entry_movement_items_details(
                         "pkgUnitCd": fetched_item.custom_packaging_unit_code,
                         "pkg": 1,
                         "qtyUnitCd": fetched_item.custom_unit_of_quantity_code,
-                        "qty": abs(item.qty),
+                        "qty": qty,
                         "itemExprDt": "",
-                        "prc": (
-                            round(int(item.basic_rate), 2) if item.basic_rate else 0
-                        ),
-                        "splyAmt": (
-                            round(int(item.basic_rate), 2) if item.basic_rate else 0
-                        ),
+                        "prc": prc,
+                        "splyAmt": prc,
                         # TODO: Handle discounts properly
                         "totDcAmt": 0,
-                        "taxTyCd": fetched_item.custom_taxation_type_code or "B",
+                        "taxTyCd": fetched_item.custom_taxation_type or "B",
```

Comment:
It's important to select the custom_taxation_type_code field as this field will always hold the actual code value documented in the etims api. Revert this.